vscode-lean4/webview/index.ts

```diff
@@ -5,7 +5,7 @@ import { Rpc } from '../src/rpc';
 
 const vscodeApi = acquireVsCodeApi();
 
-const rpc = new Rpc((m) => vscodeApi.postMessage(m));
+const rpc = new Rpc((m) => { try { vscodeApi.postMessage(m) } catch { /* ignore errors */} });
```

Comment:
Did you run into an error appearing here? Usually it is an indication of a bug elsewhere in the application. If there is one specific kind of error that is okay to ignore, we should catch only that and explain why it is okay to do so, but in general I would strongly prefer we avoid empty `catch` blocks because they tend to make it difficult to find out what has gone wrong later on.

Reply:
I'm trying to fix some random shutdown stack traces in this area that seem to happen while the tests are running after close all windows and vs is shutting down.  It is not something a user would ever see, but it does mess up the test logs.

Reply:
What do you mean by "mess up the test logs"? Do the tests fail, or is there just some extra output? In either case it sounds like we are maybe not shutting down properly which would be the real bug. I still think we shouldn't avoid empty `catch` to the extent possible.

Reply:
But your point about hiding other future errors is a good one, so I removed the try catch.